app/assets/javascripts/client/inventories/InventoryInvitation.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').factory('InventoryInvitation', ['$resource', 'UrlService', function($resource, UrlService) {
```

Comment:
Style check:  this should be using the same style we've used for all factories, controllers, and directives, in that the function for the factory and its injectables are broken out.